app/helpers/application_helper.rb

```diff
@@ -328,11 +328,6 @@ def hosts_count(resource_name = controller.resource_name)
     @hosts_count ||= HostCounter.new(resource_name)
   end
 
-  def webpack_dev_server
-    return unless Rails.configuration.webpack.dev_server.enabled
-    javascript_include_tag "#{@dev_server}/webpack-dev-server.js"
-  end
```

Comment:
Does this imply we no longer have live reloads, or did that never work in the first place?